src/components/OrderForm.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import OrderFormItem from "./OrderFormItem";
+
+class OrderForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { subtotal: "", total: "" };
+
+    this.deleteFormItem = this.deleteFormItem.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.sendOrder = this.sendOrder.bind(this);
+  }
+
+  componentDidMount() {
+    const { orders } = this.props;
+    let prices = Array.from(orders).map(el => el.price);
+    let totalPrice = prices.reduce(function(acc, item) {
+      return acc + item;
+    }, 0);
+    let priceWithDelivery = totalPrice + 2.95;
```

Comment:
It would be good to extract th delivery charge into variable and use that. That way it will be clearer what 2.95 refers from the variable name. Also, the variable could be replaced in one place if it had to be changed